ms-common/src/main/java/net/trajano/ms/spi/CacheNames.java

```diff
@@ -0,0 +1,18 @@
+package net.trajano.ms.spi;
+
+/**
+ * Cache names.
+ */
+public class CacheNames {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)